backend/ops_api/ops/migrations/0001_initial.py

```diff
@@ -312,32 +327,8 @@ class Migration(migrations.Migration):
             ],
             options={
                 "verbose_name_plural": "CANs (fiscal year)",
+                "db_table": "ops_can_fiscal_year",
                 "unique_together": {("can", "fiscal_year")},
             },
         ),
     ]
-
-    # If the old tables still exist within local Docker env or in Cloud.gov then clean them up
-    # Do not run this in unit tests - sqllite does not support CASCADE
-    if os.getenv("DJANGO_SETTINGS_MODULE") != "ops_api.django_config.settings.test":
-        operations.append(
-            migrations.RunSQL(
-                [
-                    "DROP TABLE IF EXISTS ops_site_can CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_can_funding_source CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_canfiscalyear CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_canfiscalyear_can_lead CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_contract CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_contract_cans CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_contractlineitem CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_contractlineitemfiscalyear CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_contractlineitemfiscalyearpercan CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_fundingpartner CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_fundingsource CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_person CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_person_roles CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_portfolio CASCADE;",
-                    "DROP TABLE IF EXISTS ops_site_role CASCADE;",
-                ]
-            )
-        )
```

Comment:
@johndeange, did you mean to delete this custom code?  I know we talked about pros/cons of having this, but I wasn't sure if you changed your mind or just forgot to migrate it?

Reply:
I did intend forget to bring this in when I squashed the migration file but I don't think we need it - it was run once in the dev environment and so shouldn't be needed anymore.